### PR TITLE
Add JavaScript Network Layer

### PR DESCRIPTION
**Pull request by Kikobeats**
I don't know if the JavaScript section is the more correct... what do you think?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 
 - [JavaScript](https://github.com/sorrycc/awesome-javascript)
 	- [Promises](https://github.com/wbinnssmith/awesome-promises)
+	- [Network Layer](https://github.com/Kikobeats/awesome-network-js/blob/master/README.md#awesome-javascript-network--)
```
Comment (sindresorhus) [inline readme.md]: Link to the repo, not the readme file.

**Comment by sindresorhus** [comment]
Would you mind reading the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md) :)

**Comment by sindresorhus** [comment]
> Your awesome list should have the awesome badge, a license, and contribution guidelines. Adding a code of conduct is also strongly recommended. - https://github.com/sindresorhus/awesome/blob/master/create-list.md

**Comment by sindresorhus** [comment]
And the merge conflict needs to be fixed.

**Comment by Kikobeats** [comment]
Added license → https://github.com/Kikobeats/awesome-network-js/blob/master/LICENSE.md
Added contributing → https://github.com/Kikobeats/awesome-network-js/blob/master/CONTRIBUTING.md

**Pull request closed without merging**